inspire/base/templates/records/base.html

```diff
@@ -0,0 +1,83 @@
+{#
+# This file is part of INSPIRE.
+# Copyright (C) 2015 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{% from "records/Inspire_Default_HTML_detailed_macros.tpl" import display_collections, display_current_collection, search_bar with context %}
+
+{% extends "page.html" %}
+
+{#
+{% set title = title %}
+#}
+{% block title %}
+{% endblock title %}
+
+{% block metaheader %}
+  {% if recid != -1 and config.CFG_WEBSEARCH_DETAILED_META_FORMAT %}
+  {# format_record(record, of=config.CFG_WEBSEARCH_DETAILED_META_FORMAT, ln = g.ln)|safe #}
```

Comment:
Please, have a look at what is actually being outputted here and if really need it.